extension-iap/src/java/com/defold/iap/IapGooglePlay.java

```diff
@@ -110,16 +117,54 @@ private String convertPurchase(Purchase purchase) {
         return p.toString();
     }
 
-    private JSONObject convertSkuDetails(SkuDetails skuDetails) {
+    private JSONArray convertSubscriptionOfferPricingPhases(SubscriptionOfferDetails details) {
+        JSONArray a = new JSONArray();
+        try {
+            List<PricingPhase> pricingPhases = details.getPricingPhases().getPricingPhaseList();
+            for (PricingPhase pricingPhase : pricingPhases) {
+                JSONObject o = new JSONObject();
+                o.put("price_string", pricingPhase.getFormattedPrice());
+                o.put("price", pricingPhase.getPriceAmountMicros() * 0.000001);
+                o.put("currency_code", pricingPhase.getPriceCurrencyCode());
+                o.put("billing_period", pricingPhase.getBillingPeriod());
+                o.put("billing_cycle_count", pricingPhase.getBillingCycleCount());
+                o.put("recurrence_mode", pricingPhase.getRecurrenceMode());
+                a.put(o);
```

Comment:
Maybe we should add info in doc about new fields in response?

Reply:
Added!